src/components/api/axios.js

```diff
@@ -0,0 +1,5 @@
+import axios from "axios";
+
+export const imageURL = axios.get(
```

Comment:
`imageURL` is not a worthy name.

It sounds more like a string than an axiosInstance